uid2_client/input_util.py

```diff
@@ -0,0 +1,104 @@
+import hashlib
+import base64
+
+
+def is_phone_number_normalized(phone_number):
```

Comment:
i know there's no unit tests for java sdk - could you please start creating a few unit tests for phone/email normalisation? 

for test cases, see https://unifiedid.com/docs/getting-started/gs-normalization-encoding and https://github.com/IABTechLab/uid2-operator/blob/master/src/test/java/com/uid2/operator/InputNormalizationTest.java

Reply:
there's Java normalization tests, see hashedGmail() and unnormalizedOrInvalidPhone()

Reply:
Adding some tests. Should this java logic not be in shared?

Reply:
Fairly question but we just haven't gotten around to it how best to share these logics in Java (getting uid2-client-java to depend on uid2-shared is a bit heavy=handed but then creating another small shared lib is also a hassle)

Reply:
This is UID2-1206 but it's a controversial ticket

Reply:
Added tests for normalization of emails and phone numbers